data/man/corecollector_conf.5.scd

```diff
@@ -1,15 +1,15 @@
 corecollector.conf(5)
 
 # NAME
-corecollector.conf - configuration for corecollector, a coredump collector
+corecollector.conf - configuration for corecollector
 
 # DESCRIPTION
 The *corecollector.conf* file is used to configure corecollector. As of now it
-supports the following, not case sensitive, keys:
+supports the following, not case sensitive, keywords:
 
 *compression*
 	This sets what compression algorithm should be used for compression.
-	As of now none are supported.
+	Zlib is supported, while no compression is the default.
```

Comment:
Seems like scdoc turns this into two spaces: `compression.  Zlib is supported, while no compression is the`

Reply:
This is how it looked like before the changes, too. So I think this is a bug with scdoc. (Don't start new sentences on indented new lines unless there's whitespace above)

I double checked with `[man 5 sway](https://github.com/swaywm/sway/blob/master/sway/sway.5.scd#L204)` and see the same behaviour.

Reply:
This is also present on `corectl.scd` `errors. 0 also prints warnings`

Reply:
Ah okay, thanks for checking.